projects/plugins/boost/app/modules/cache/pre-wordpress/Request.php

```diff
@@ -148,6 +143,11 @@ public function is_cacheable() {
 			return false;
 		}
 
+		if ( $this->is_url_excluded() ) {
+			Logger::debug( 'Url excluded, not cached!' ); // phpcs:ignore -- This is a debug message
```

Comment:
I think the phpcs:ignore was left there by mistake.
```suggestion
			Logger::debug( 'Url excluded, not cached!' );
```